File: 프로그래머스/0/181912. 배열 만들기 5/배열 만들기 5.js
function solution(intStrs, k, s, l) {
    let answer = [];
    for (let i = 0; i< intStrs.length; i++) {
        let temp = parseInt(intStrs[i].slice(s, s + l));
        if (temp > k) {
            answer.push(temp);
        }
    }
    return answer;
}

File: 프로그래머스/0/181912. 배열 만들기 5/배열 만들기 5.py
def solution(intStrs, k, s, l):
    answer = []
    for int_str in intStrs:
        temp_str = int_str[s : s + l]
        temp_num = int(temp_str)

        if temp_num > k:
            answer.append(temp_num)          
    return answer